packages/polymer-legacy-adapter/test/vaadin-template-renderer.test.js

```diff
@@ -119,6 +119,17 @@ describe('vaadin-template-renderer', () => {
     expect(spy.calledOnce).to.be.true;
   });
 
+  it('should include model in the event', () => {
+    const host = fixtureSync(`<mock-component-host></mock-component-host>`);
+    const component = host.$.component;
+    const button = component.$.content.querySelector('button');
+    const spy = sinon.spy(host, 'onClick');
+
+    click(button);
+
+    expect(spy.getCall(0).args[0].model).to.be.ok;
```

Comment:
Maybe it is better to check whether the `model` argument refers to the proper template instance.

```suggestion
    expect(spy.getCall(0).args[0].model).to.equal(component.$.content.__templateInstance);
```